src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/AprilTagCamera.java

```diff
@@ -13,30 +13,31 @@
 
 /**
  * An april tag camera is a class that provides the robot's pose, from a camera.
+ * An april tag is like a 2D barcode
  */
 public class AprilTagCamera {
     protected final String name;
     private final RobotPoseSourceInputsAutoLogged inputs = new RobotPoseSourceInputsAutoLogged();
     private final Transform3d robotCenterToCamera;
     private final AprilTagCameraIO aprilTagCameraIO;
     private double lastUpdatedTimestamp;
-    private final double solvePNPTranslationsStdExponent, assumedRobotPoseTranslationsStdExponent, solvePNPThetaStdExponent;
+    private final double solvePNPTranslationsStandardDeviationsExponent, assumedRobotPoseTranslationsStandardDeviationsExponent, solvePNPThetaStandardDeviationsExponent;
     private Pose2d robotPose = null;
 
     /**
-     * @param robotPoseSourceType                        the type of camera
-     * @param name                                       the camera's name
-     * @param robotCenterToCamera                        the transform of the robot's origin point to the camera
-     * @param solvePNPTranslationsStdExponent            the translation exponent for solve PNP
-     * @param solvePNPThetaStdExponent                   the theta exponent for solve PNP
-     * @param assumedRobotHeadingTranslationsStdExponent the translation exponent for the assumed robot heading pose calculation
+     * @param robotPoseSourceType                                       the type of camera
```

Comment:
Its common practice to write at the start on the general explanation `Constructs a new AprilTagCamera.`